web-src/src/common/components/history/executions-log-table.vue

```diff
@@ -44,38 +62,64 @@ export default {
     }
   },
 
+  data() {
+    return {
+      filteredRows: this.rows ? [...this.rows] : [],
```

Comment:
Instead of using data+watch, it's better to use computed property.

Reply:
Got it. Just very new to vue, so learning.